statistics/handle/handle.go

```diff
@@ -62,6 +63,8 @@ const (
 
 // Handle can update stats info periodically.
 type Handle struct {
+	gpool *gp.Pool
```

Comment:
Please add some comments to describe what this Pool is used for in `StatsHandle`.

Reply:
Done